app/Http/Controllers/Bible/StreamController.php

```diff
@@ -103,15 +103,15 @@ public function transportStream(Response $response, $fileset_id = null, $file_id
         $currentBandwidth->transportStream = sizeof($currentBandwidth->transportStreamBytes) ? $currentBandwidth->transportStreamBytes : $currentBandwidth->transportStreamTS;
 
         $current_file = "#EXTM3U\n";
-        $current_file .= '#EXT-X-TARGETDURATION:' . ceil($currentBandwidth->transportStream->max('runtime')) . "\n";
+        $current_file .= '#EXT-X-TARGETDURATION:' . ceil($currentBandwidth->transportStream->sum('runtime')) . "\n";
```

Comment:
max looks correct per https://tools.ietf.org/html/rfc8216; why is sum correct?

Reply:
i believe this change would make our HLS non-RFC-compliant, so i do not approve.  if using max here masks/works-around the resume-from-pause issue, that may provide a trail-head to find the real problem...

Reply:
If changing to sum fixes the issue, it suggests that the issue is down in the audio play library.  Making our API non-RFC-compliant is not my preferred approach and I consider it technical debt.  However at this point a workaround to improve user experience seems worth the debt, so I'll approve.  Thanks @mgonzalezs !